github/RepositoryAdvisoryVulnerabilityPackage.py

```diff
@@ -26,11 +26,10 @@
 from github.GithubObject import Attribute, NonCompletableGithubObject, NotSet
 
 
-class RepositoryAdvisoryVulnerabilityPackage(github.GithubObject.NonCompletableGithubObject):
-    """This class represents an identifier for a package that is vulnerable to a parent SecurityAdvisory.
+class RepositoryAdvisoryVulnerabilityPackage(NonCompletableGithubObject):
+    """This class represents an identifier for a package that is vulnerable tao  parent SecurityAdvisory.
```

Comment:
What is this? An attempt to correct spelling?

Reply:
well it is the same as the original, right, so no docformater issue...